packages/ts-config/tsconfig.json

```diff
@@ -5,15 +5,12 @@
 		"alwaysStrict": true,
 		"declaration": true,
 		"declarationMap": true,
-		"emitDecoratorMetadata": true,
 		"esModuleInterop": true,
-		"experimentalDecorators": true,
-		"forceConsistentCasingInFileNames": true,
 		"importHelpers": true,
-		"importsNotUsedAsValues": "error",
+		"verbatimModuleSyntax": true,
 		"incremental": true,
 		"lib": ["esnext"],
-		"module": "CommonJS",
+		"module": "Node16",
 		"moduleResolution": "Node",
```

Comment:
Should also be Node16!

Reply:
No actually, setting both causes tonnes of these errors again (same as just setting `module` to `CommonJS`)

<img width="797" alt="CleanShot 2023-03-17 at 22 29 45@2x" src="https://user-images.githubusercontent.com/4019718/226057048-32a8103c-6220-4c50-9629-a2e0b6bb59a4.png">

Note also how by default it is specifically `Node` when `module` is set to `node16`: https://www.typescriptlang.org/tsconfig#moduleResolution

Reply:
That sounds like a bug on TS side because I recall them saying you need both!

Reply:
Contrary, they dont. They don't even mention `moduleResolution` on the blog post where they announced `node16`: https://devblogs.microsoft.com/typescript/announcing-typescript-4-7/.

Even if you include the typo'd `moduleresoluton`, that is still only 1 hit and doesn't talk about the new option for that configuration. 

Furthermore, the reference page for `moduleResolution` also doesn't list `node16`: https://www.typescriptlang.org/docs/handbook/module-resolution.html